tools/serverpod_cli/lib/src/create/create.dart

```diff
@@ -94,363 +124,423 @@ Future<void> performCreate(
   var awsName = name.replaceAll('_', '-');
   var randomAwsId = math.Random.secure().nextInt(10000000).toString();
 
-  var projectDir = Directory(p.join(Directory.current.path, name));
-  if (projectDir.existsSync()) {
+  var serverpodDirs = ServerpodDirectories(
+    projectDir: Directory(p.join(Directory.current.path, name)),
+    name: name,
+  );
+  if (serverpodDirs.projectDir.existsSync()) {
     log.error('Project $name already exists.');
-    return;
+    return false;
   }
 
-  log.info(
-    'Creating project $name.',
-    style: const TextLogStyle(newParagraph: true),
-  );
+  if (template == ServerpodTemplateType.server) {
+    log.info(
+      '🚀 Creating Serverpod project "$name"',
+      style: const TextLog(type: TextLogType.init),
+    );
+  } else if (template == ServerpodTemplateType.module) {
+    log.info(
+      '📦 Creating Serverpod module "$name"',
+      style: const TextLog(type: TextLogType.init),
+    );
+  }
 
-  log.debug(
-    'Creating directory: ${projectDir.path}',
-    style: const TextLogStyle(
-      type: AbstractStyleType.bullet,
-    ),
-  );
-  projectDir.createSync();
+  bool success = await log.progress(
+      'Setting up project structure',
+      () => Future(() {
+            _createProjectDirectories(template, serverpodDirs);
+            return true;
+          }),
+      newParagraph: true);
 
-  var serverDir = Directory(p.join(projectDir.path, '${name}_server'));
-  log.debug(
-    'Creating directory: ${serverDir.path}',
-    style: const TextLogStyle(
-      type: AbstractStyleType.bullet,
-    ),
-  );
-  serverDir.createSync();
+  if (template == ServerpodTemplateType.server) {
+    success &= await log.progress(
+        'Bootstrapping project files',
```

Comment:
"Bootstrapping" feels technical and it hard to understand. "Writing project files." is more clear.